hack/build_proxy.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+: "${ISTIO_GITHUB_REPO_URL:=https://github.com/maistra/istio.git}"
+: "${ISTIO_GITHUB_BRANCH:=maistra-3.0}"
+: "${PROXY_GITHUB_REPO_URL:=https://github.com/maistra/proxy.git}"
+: "${PROXY_GITHUB_BRANCH:=maistra-3.0-pre}"
+
+function download_envoy_binary() {
+  if [ $# -ne 1 ]; then
+    echo "Usage: download_envoy_binary workdir"
+    exit 1
+  fi
+
+  local workdir="${1}"
+
+  git clone --single-branch -b "${ISTIO_GITHUB_BRANCH}" "${ISTIO_GITHUB_REPO_URL}" "${workdir}/istio"
```

Comment:
use `--depth=1` in all git clone calls

Reply:
Actually, this script will run - as part of a post-submit job) on the latest istio tree. So, there's no need to git clone maistra/istio.